README.markdown

```diff
@@ -399,31 +454,61 @@ Manages multiple values within the same INI setting.
 
 ##### `ensure`
 
-Specifies whether the subsetting should be present. Valid options: 'present' and 'absent'. Default value: 'present'.
+Specifies whether the subsetting should be present. 
+
+Valid options: 'present' and 'absent'. 
+
+Default value: 'present'.
 
 ##### `key_val_separator`
 
-*Optional.* Specifies a string to use between setting name and value (e.g., to determine whether the separator includes whitespace). Valid options: a string. Default value: ' = '.
+*Optional.* 
+
+Specifies a string to use between setting name and value (e.g., to determine whether the separator includes whitespace). 
+
+Valid options: a string. 
+
+Default value: ' = '.
 
 ##### `path`
 
-*Required.* Specifies an INI file containing the subsetting to manage. Valid options: a string containing an absolute path.
+*Required.* 
+
+Specifies an INI file containing the subsetting to manage. 
+
+Valid options: a string containing an absolute path.
 
 ##### `quote_char`
 
-*Optional.* The character used to quote the entire value of the setting. Valid values are '', '"', and "'". Defaults to ''. Valid options: '', '"' and "'". Default value: ''.
+*Optional.* 
+
+The character used to quote the entire value of the setting. Valid values are '', '"', and "'". Defaults to ''. 
+
+Valid options: '', '"' and "'". 
+
+Default value: ''.
 
 ##### `section`
 
-*Optional.* Designates a section of the specified INI file containing the setting to manage. To manage a global setting (at the beginning of the file, before any named sections) enter "". Defaults to "". Valid options: a string.
+*Optional.* 
+
+Designates a section of the specified INI file containing the setting to manage. To manage a global setting (at the beginning of the file, before any named sections) enter "". Defaults to "". 
```

Comment:
This parenthetical statement probably shouldn't be parenthetical. I think it means that you have to put global settings at the beginning of the file before any named sections, so you should rewrite this to make that more clear.